source/development/contributing.rst

```diff
@@ -493,10 +524,10 @@ file.
 Branch Protection
 *****************
 
-All repositories' ``main`` branch must be configured with a set of protection rules that aim at
-ensuring some of the rules defined in :ref:`contribution workflow<contribution_workflow>`. In the
-repository's ``Settings -> Branches`` menu, the protection rules must be created with the following
-options:
+All repositories' ``main`` and ``wip/`` (base branch of a split of PRs) branches must be configured
```

Comment:
Are you proposing that we should reserve "wip/" for split PRs?

Reply:
That's a good point. Taking into consideration that we are using as the target branch for splitting PRs and protecting it to avoid merging without reviews, maybe we should reserve it. What do you think @josecm ?

Reply:
Well, I was thinking that protecting the branch will be a problem, but now I realize it isn't. So, yeah, I don't think we need to lock the tag.

Reply:
The name needs to be unique to avoid the need to always add the base branch as a protected branch. Nevertheless, we don't need to lock it only for splitting PRs.

Reply:
Now I'm confused.
If we apply the rules to all wip/ branches won't we be limiting the non-"split PR" usage of the wip/ branches?

Reply:
I agree with @DavidMCerdeira. This paragraph should only mention obligation for the `main`branch, but the "Split PR" section should point to this section mentioning that for that case the `wip` branch should be protected.

Reply:
Fixed.